linksharing/handler.go

```diff
@@ -60,10 +70,16 @@ func NewHandler(log *zap.Logger, config HandlerConfig) (*Handler, error) {
 		return nil, err
 	}
 
+	mapper, err := objectmap.NewIPDB("./static/maxMindDB.mmdb")
```

Comment:
It is preferable to load the `maxmind` database file path from config. Maxmind databases are updated very often.

Reply:
Or you can keep this and enforce to always use the same maxmind database name.

Reply:
done